LayoutTestBase/UIViewHelpers/UIView+LYTFrameComparison.h

```diff
@@ -25,6 +25,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)lyt_before:(UIView *)otherView NS_SWIFT_NAME(before(_:));
 
+/**
+ Version of lyt_before that takes a fromCenter param. If fromCenter is YES, the comparison starts from the center of the view rather than the
+ edge, allowing an overlap of up to half of the view. So for example, if this view is at x=0 with a width=10, and another view is at x=6, width=10,
+ the function will return YES if fromCenter==YES, NO otherwise. The function honors the epsilon vlaue and right-to-left.
+ 
+ \param otherView The view you want to compare to. It does not need to have the same superview, but MUST share some ancestor view.
+ \param fromCenter If YES, allow an overlap up to the center of otherView. If NO, behavior is the same as without the param
+ \returns True if the view is laid out before another view, allowing for an overlap up to the center if fromCenter is YES.
+ */
+- (BOOL)lyt_before:(UIView *)otherView fromCenter:(BOOL)fromCenter NS_SWIFT_NAME(before(_:_:));
```

Comment:
Don't you want the Swift name to be `before(_:fromCenter:)`? I haven't done this from ObjC, so correct me if I'm wrong, and the syntax you have automatically takes the second param name.

Reply:
Fixed